sql/sqltool/tool.go

```diff
@@ -351,6 +356,23 @@ func (d *FlywayDir) Files() ([]migrate.File, error) {
 	return ret, nil
 }
 
+// Checksum implements Dir.Checksum. By default, it calls Files() and creates a checksum from them.
+func (d *FlywayDir) Checksum() (migrate.HashFile, error) {
```

Comment:
```suggestion
func (d *FlywayDir) Checksum() (migrate.HashFile, error) {
	if check, ok d.FS.(migrate.Dir); ok {
		return check.Checksum()
	}
```

Reply:
?

Reply:
respect the checksum implement in .FS if existing